src/doc/oiiotool.rst

```diff
@@ -119,10 +119,10 @@ contents of an expression may be any of:
 * *imagename.metadata*
 
   The named metadata of an image.
-  
+
   If there is no metadata whose name matches, the expression will not have any
   substitution made and an error will be issued.
-  
+
   The *imagename* may be one of:
```

Comment:
Something odd happened here, your editor seems to have introduced whitespace changes all through this document.

Reply:
This is embarrassing, apologies Larry! 

Was looking at this and I think that's still affected, will take a look again later.

Reply:
Hi Larry, 

I believe the whitespace issue is walked back, let me know if those still look wrong?
Seems like the editor was trimming them, but I've verified with an older commit hash of the file that this should be what we started with.

Also the string comparison suggestion should be there as well!